src/colmap/feature/types.h

```diff
@@ -84,15 +84,15 @@ typedef Eigen::Matrix<float, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>
 
 struct FeatureMatch {
   FeatureMatch()
-      : point2D_idx1(kInvalidPoint2DIdx), point2D_idx2(kInvalidPoint2DIdx) {}
+      : point2D_idx1(kInvalidPoint2DId), point2D_idx2(kInvalidPoint2DId) {}
   FeatureMatch(const point2D_t point2D_idx1, const point2D_t point2D_idx2)
       : point2D_idx1(point2D_idx1), point2D_idx2(point2D_idx2) {}
 
   // Feature index in first image.
-  point2D_t point2D_idx1 = kInvalidPoint2DIdx;
```

Comment:
1. I think that it should really be an index and not an ID since 2D points are stored in an ordered list while image/camera/points3D IDs are not guaranteed to be consecutive.
2. This would unnecessarily break many dependencies (incl. GLOMAP) so IMHO it's not worth it.
3. Even then, this would need to be changed in a separate PR.

Reply:
Thanks. This makes sense. I have reverted it.